src/features/account/index.tsx

```diff
@@ -0,0 +1,86 @@
+import styles from './styles.module.scss';
+import {UserData} from '../shared/types';
+import {Typography, Box} from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import {useState} from 'react';
+import UpdateBasicInfoDrawer from './update-basic-info-drawer';
+import PrimaryButton from '../shared/components/primary-button';
+import ChangePasswordDrawer from './change-password-drawer';
+
+type Props = {
+  user: UserData;
+};
+
+export default function Account({user}: Props) {
+  const [userData, setUserData] = useState<UserData>(user);
+  const [showBasicInfoDrawer, setShowBasicInfoDrawer] = useState(false);
+  const [showChangePasswordDrawer, setShowChangePasswordDrawer] = useState(false);
+
+  const handleOpenBasicInfoDrawer = () => {
+    setShowBasicInfoDrawer(true);
+  };
+
+  const handleCloseBasicInfoDrawer = () => {
+    setShowBasicInfoDrawer(false);
+  };
+
+  const handleOpenChangePasswordDrawer = () => {
+    setShowChangePasswordDrawer(true);
+  };
+
+  const handleCloseChangePasswordDrawer = () => {
+    setShowChangePasswordDrawer(false);
+  };
+
+  return (
+    <>
+      <div className={styles.accountContainer}>
```

Comment:
Could you use a Box here? I like to use everything available from MUI. I think the page will look more consistent,